modules/lmdbbackend/meson.build

```diff
@@ -6,4 +6,4 @@ module_extras = files(
   'lmdbbackend.hh',
 )
 
-module_deps = [deps, dep_lmdb_safe, dep_lmdb, dep_boost_serialization]
+module_deps = [deps, dep_lmdb_safe, dep_lmdb, dep_boost_serialization, dep_systemd]
```

Comment:
tihs seems weird

Reply:
I thought so. When I was debugging the other issue and looked at the autotools `Makefile.am` for the LMDB backend, I found the following:
```
liblmdbbackend_la_LIBADD = $(LMDB_LIBS) $(BOOST_SERIALIZATION_LIBS) $(SYSTEMD_LIBS)
```
And that's why I added it here.

Reply:
It is, in fact, needed. From `lmdbbackend.cc`:
```
#ifdef HAVE_SYSTEMD
  /* A schema migration may take a long time. Extend the startup service timeout to 1 day,
   * but only if this is beyond the original maximum time of TimeoutStartSec=.
   */
  sd_notify(0, "EXTEND_TIMEOUT_USEC=86400000000");
#endif
```

Reply:
oh right!